plugins/repository-gcs/build.gradle

```diff
@@ -293,29 +293,28 @@ testClusters {
  * We only use a small amount of data in these tests, which means that the resumable upload path is not tested. We add
  * an additional test that forces the large blob threshold to be small to exercise the resumable upload path.
  */
-task largeBlobYamlRestTest(type: RestIntegTestTask) {
-  dependsOn bundlePlugin
+def largeBlobYamlRestTest = tasks.register("largeBlobYamlRestTest", RestIntegTestTask) {
+  dependsOn "bundlePlugin"
   if (useFixture) {
-    dependsOn createServiceAccountFile
+    dependsOn "createServiceAccountFile"
   }
-    SourceSetContainer sourceSets = project.getExtensions().getByType(SourceSetContainer.class);
-    SourceSet yamlRestTestSourceSet = sourceSets.getByName(YamlRestTestPlugin.SOURCE_SET_NAME)
-    setTestClassesDirs(yamlRestTestSourceSet.getOutput().getClassesDirs())
-    setClasspath(yamlRestTestSourceSet.getRuntimeClasspath())
-}
+  SourceSetContainer sourceSets = project.getExtensions().getByType(SourceSetContainer.class);
+  SourceSet yamlRestTestSourceSet = sourceSets.getByName(YamlRestTestPlugin.SOURCE_SET_NAME)
+  setTestClassesDirs(yamlRestTestSourceSet.getOutput().getClassesDirs())
+  setClasspath(yamlRestTestSourceSet.getRuntimeClasspath())
 
-check.dependsOn largeBlobYamlRestTest
+  // We have to wait for configure the cluster here as it might not have been created otherwise yet.
```

Comment:
By this you just mean that we move this config _inside_ the task configuration block as that guarantees the task has been created, and subsequently, the corresponding testcluster will exist as well?

Reply:
exactly